src/openms/source/FORMAT/HANDLERS/MzMLHandler.cpp

```diff
@@ -692,25 +692,27 @@ namespace OpenMS::Internal
       constexpr XMLCh s_external_spectrum_id[] = { 'e','x','t','e','r','n','a','l','S','p','e','c','t','r','u','m','I','D' , 0};
       // constexpr XMLCh s_default_source_file_ref[] = { 'd','e','f','a','u','l','t','S','o','u','r','c','e','F','i','l','e','R','e','f' , 0};
       constexpr XMLCh s_scan_settings_ref[] = { 's','c','a','n','S','e','t','t','i','n','g','s','R','e','f' , 0};
-      String tag = sm_.convert(qname);
-      open_tags_.push_back(tag);
+      
+      
+      open_tags_.push_back(sm_.convert(qname));
+      const String& tag = open_tags_.back();
```

Comment:
I guess you checked that this is still thread safe etc. (I would assume so because otherwise we would run into issues with open_tags_ anyway)

Reply:
this code is not parallelized, so not relevant
(parallelization only happens in chunks in populateSpectra_ etc)